zipline/finance/performance/tracker.py

```diff
@@ -191,6 +188,10 @@ def __init__(self, sim_params, env=None):
         self.account_needs_update = True
         self._account = None
 
+        self._known_asset_sids = set()
```

Comment:
What are these used for?  It seems like there's a lot of state tracking here that's being duplicated in the `AssetFinder`